pages/common/mysql.md

```diff
@@ -8,7 +8,11 @@
 
 - Connect to a database using credentials
 
-`mysql -u {{user}} -p{{password}} {{database_name}}`
+`mysql -h {{database_host}} -u {{user}} -p{{password}} {{database_name}}`
+
+- Connect to a database on a different host. MySQL will prompt for a password if none is supplied.
```

Comment:
The wording "on a different host" is clunky in comparison to the previous command. It seems it tries to contradict, but that's not the case. Perhaps "Connect by entering the password interactively"?